cdk/tsconfig.json

```diff
@@ -1,12 +1,5 @@
 {
-  "extends": "@guardian/tsconfig/tsconfig.json",
-  "compilerOptions": {
-    "module": "CommonJS"
-  },
```

Comment:
Is removing `"module": "CommonJS"` the reason why we need to specify file extensions in imports now?

I'm slightly confused at how this builds as the `.js` files we reference in the imports shouldn't exist!